usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -796,7 +796,7 @@ function make_pxelinux_config {
     echo "say ----------------------------------------------------------"
 
     # start with optional rear http entry if specified
-    if [[ ! -z $PXE_HTTP_URL ]] ; then    
+    if [[ ! -z $PXE_HTTP_DOWNLOAD_URL ]] ; then    
```

Comment:
```suggestion
    if [[ "$PXE_HTTP_DOWNLOAD_URL" ]] ; then    
```
we prefer positive logic and have to quote